src/pyflakes/format.rs

```diff
@@ -20,3 +26,110 @@ impl fmt::Display for FormatParseError {
         write!(f, "{message}")
     }
 }
+
+pub(crate) struct FormatSummary {
+    pub autos: HashSet<usize>,
+    pub indexes: HashSet<usize>,
```

Comment:
I'll likely change these to `FxHashSet` too, unless they're intentionally not using that!

Reply:
Not intentional, just that I was unsure about the benchmarks for various data types, and the initial review pointed to the stringy sets. Fixed now.